indico/modules/events/surveys/templates/submission_preview.html

```diff
@@ -0,0 +1,69 @@
+{% extends 'layout/base.html' %}
+
+{% block title %}
+    {% trans %}Survey{% endtrans %}
+{% endblock %}
+
+{% block subtitle %}
+    {% trans submission_id=submission.id %}Preview of the submission #{{ submission_id }}{% endtrans %}
+{% endblock %}
+
+{% block content %}
+    <div class="page-content-description">
+        {% set submission_dt = submission.submitted_dt | format_datetime %}
+        <p>
+            {% if submission.user %}
+                {% trans dashboard=url_for('users.user_dashboard', submitter), submitter_name=submission.user.full_name %}
```

Comment:
only Indico admins can access the dashboard for another user. so linking there is not a good ideafor now